crates/rustc_codegen_spirv/src/builder/spirv_asm.rs

```diff
@@ -328,10 +366,32 @@ impl<'cx, 'tcx> Builder<'cx, 'tcx> {
                 }
                 return;
             }
-            _ => {
+
+            op => {
                 self.emit()
                     .insert_into_block(dr::InsertPoint::End, inst)
                     .unwrap();
```

Comment:
Still would be nice to not shove everything into the same block, including terminators and labels, but whatever I guess!